app.test.js

```diff
@@ -101,14 +100,61 @@ describe('Server', () => {
       expect(revisedProject).toEqual(expectedResult);
     });
 
```

Comment:
Below is one of the sad path tests (PATCH) with which I was having trouble.
Would it be alright to just expect the response status to be 404?